### PR TITLE
Improve routing debug page

### PR DESCRIPTION
**Pull request by atailouloute**
Some small improvement for the routing debug doc (http://symfony.com/doc/2.7/routing/debug.html)

### CODE REVIEW

**Review by javiereguiluz — COMMENTED** [review]
@atailouloute thanks for this improvement. However, I'd like to ask some changes:

1) The Console Style Guide which changed the look of commands output was introduced in 2.8 (https://symfony.com/blog/new-in-symfony-2-8-console-style-guide) so maybe we shouldn't use it here in 2.7 branch.
2) We usually don't document the options of console commands (such as `--show-controllers`) because they are hard to maintain in the docs.

In any case, please don't make any change until we know the opinion of the other doc maintainers. Thanks!

**Comment by xabbuh** [comment]
I agree with Javier on both issues he raised.

File: routing/debug.rst [hunk 1]
```diff
@@ -21,12 +21,15 @@ your application:
...
-    blog_show             ANY       /blog/{slug}
+------------------ -------- -------- ------ ----------------------------------------------
+ Name               Method   Scheme   Host   Path
+------------------ -------- -------- ------ ----------------------------------------------
+ homepage           ANY      ANY      ANY    /
+ contact            GET      ANY      ANY    /contact
+ contact_process    POST     ANY      ANY    /contact
+ article_show       ANY      ANY      ANY    /articles/{_locale}/{year}/{title}.{_format}
+ blog               ANY      ANY      ANY    /blog/{page}
+ blog_show          ANY      ANY      ANY    /blog/{slug}
```
Comment (wouterj) [inline routing/debug.rst]: This should be all indented with 4 spaces, in order for it to be included in the code block defined on line 22.

**Review by wouterj — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Thanks @atailouloute.

**Pull request closed without merging**